lib/matplotlib/lines.py

```diff
@@ -1121,19 +1113,16 @@ def set_linestyle(self, ls):
         if isinstance(ls, str):
             if ls in [' ', '', 'none']:
                 ls = 'None'
-
             _api.check_in_list([*self._lineStyles, *ls_mapper_r], ls=ls)
             if ls not in self._lineStyles:
                 ls = ls_mapper_r[ls]
             self._linestyle = ls
         else:
             self._linestyle = '--'
-
-        # get the unscaled dashes
-        self._us_dashOffset, self._us_dashSeq = _get_dash_pattern(ls)
-        # compute the linewidth scaled dashes
-        self._dashOffset, self._dashSeq = _scale_dashes(
-            self._us_dashOffset, self._us_dashSeq, self._linewidth)
+        self._us_dash_pattern = _get_dash_pattern(ls)  # unscaled dashes
```

Comment:
While at it, 
```suggestion
        self._unscaled_dash_pattern = _get_dash_pattern(ls)
```
instead  of commenting every usage with "unscaled"

Reply:
sure.